public/app/features/dashboard/components/PubdashFooter/PubdashFooter.tsx

```diff
@@ -4,20 +4,40 @@ import React from 'react';
 import { GrafanaTheme2, colorManipulator } from '@grafana/data';
 import { useStyles2 } from '@grafana/ui';
 
+export interface PublicDashboardFooter {
+  disable: boolean;
+  text: string;
+  logo: string;
+  link: string;
+}
+
 export const PubdashFooter = function () {
   const styles = useStyles2(getStyles);
+  const conf = getPubdashFooterConfig();
 
-  return (
+  return conf.disable ? null : (
     <div className={styles.footer}>
       <span className={styles.logoText}>
-        <a href="https://grafana.com/" target="_blank" rel="noreferrer noopener">
-          powered by Grafana <img className={styles.logoImg} alt="" src="public/img/grafana_icon.svg"></img>
+        <a href={conf.link} target="_blank" rel="noreferrer noopener">
+          {conf.text} <img className={styles.logoImg} alt="" src={conf.logo}></img>
         </a>
       </span>
     </div>
   );
 };
 
+export function setPubdashFooterConfigFn(fn: typeof getPubdashFooterConfig) {
+  getPubdashFooterConfig = fn;
+}
```

Comment:
we should use redux for this. I understand it's already coded in this way but it's not good

Reply:
Wonder how would you achieve this with Redux? The current system has been built to allow hooking up components and functions from Enterprise into Grafana and storing those in Redux is not an option.

Reply:
you're right. my bad

Reply:
Naming here as well.

We're actively trying to remove the use of redux where possible in the frontend codebase

Reply:
why are we trying to remove redux for the frontend? @jalevin